tasks/install.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Install dependencies
```

Comment:
The only dependency is on selinux packages and those are handled by `tasks/selinux.yml` so this is no longer needed.